.circleci/config.yml

```diff
@@ -88,9 +88,12 @@ jobs:
       - run:
           name: Install infection
           command: composer require infection/infection:dev-master # Currently only master has https://github.com/infection/infection/pull/1106 merged.
+      - run:
+          name: Remove extra large coverage reports
+          command: find build/logs/coverage-xml -type f -size +100M -print -delete
       - run:
           name: Mutation coverage testing
-          command: php -d memory_limit=16G vendor/bin/infection --coverage=build/phpunit --no-progress --only-covered --threads=2
+          command: php -d memory_limit=4G vendor/bin/infection --coverage=build/logs --no-progress --only-covered --threads=2
```

Comment:
Psalm own `phpunit.xml` uses `build/logs` so if I may suggest to stick with this standard of sorts.